src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  gap: $base-gap;
+  width: ($square-size * 7) + ($base-gap * 6);
+  padding: 10px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    height: $square-size;
+    width: $square-size;
+    background-color: $square-color;
+    border: 1px solid black;
+    transition: 500ms;
+
+    &::before {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin: auto;
+    }
+
+    &:hover {
+      background-color: $square-color-hover;
+      transform: translateY(-20px);
+    }
```

Comment:
The `transform` property is used here to move the calendar cell up on hover. It's a good practice to also include `transition` property for the `transform` to make the movement smooth. You've already included `transition: 500ms;` in the `.calendar__day` class, but it would be better to specify which property you are transitioning, like so: `transition: transform 500ms;`